pkgs/development/libraries/re2/default.nix

```diff
@@ -1,16 +1,14 @@
 { lib
 , stdenv
 , fetchFromGitHub
-, nix-update-script
-
-# for passthru.tests
-, bazel
+, cmake
+, ninja
 , chromium
 , grpc
 , haskellPackages
 , mercurial
-, ninja
-, python3
+, python3Packages
+, enableShared ? stdenv.hostPlatform.isStatic
```

Comment:
Why expose this in the signature? The `makeStaticLibraries` adapter already adds the CMake flag for `pkgsStatic`.

Reply:
I saw it in a couple of places. You are correct, it does enable it [itself](https://github.com/NixOS/nixpkgs/blob/master/pkgs/stdenv/adapters.nix#L88). Fixed.

Reply:
Actually, I've now change it to
```nix
cmakeFlags = lib.optional (!stdenv.hostPlatform.isStatic) "-DBUILD_SHARED_LIBS:BOOL=ON";
```
because, apparently, the non-`pkgsStatic` version doesn't build shared libraries by default for some reason and this will prevent us from linking against these libraries.